accname/name/comp_text_node.html

```diff
@@ -13,23 +13,139 @@
 
 <p>Tests the <a href="https://w3c.github.io/accname/#comp_text_node">#comp_text_node</a> portions of the AccName <em>Name Computation</em> algorithm.</p>
 
-<!-- I'm not certain whether #comp_text_node requires a lot of testing outside of the #comp_name_from_content contexts, -->
-<!-- but I did think of one example where text node versus comment node may make a difference when joining text nodes with a space vs innerText. -->
+<!--
 
-<!-- Skipped (class="ex" removed) until https://github.com/w3c/accname/issues/193 is resolved -->
-<h2 class="ex-skipped" data-expectedlabel="heading label" data-testname="heading with text/comment/text nodes, no space">
-  heading<!-- with non-text node splitting concatenated text nodes -->label<!-- [sic] no extra spaces around first comment -->
-</h2>
+  Note: some overlap with the tests in:
+  - /accname/name/comp_label.html
+  - /accname/name/comp_name_from_content.html
+
+-->
+
+<h1>text/element/text nodes, no space</h1>
+<span role="button" tabindex="0" class="ex" data-expectedlabel="buttonlabel" data-testname="span[role=button] with text/element/text nodes, no space">button<span></span>label</span>
+<div role="heading" class="ex" data-expectedlabel="headinglabel" data-testname="div[role=heading] with text/element/text nodes, no space">heading<span></span>label</div>
```

Comment:
The test looks good, somewhat related observation:

I noticed that HTML validation passes for this role="heading" without aria-level however, it's currently [a MUST that aria-level be provided](https://w3c.github.io/aria/#heading). Should this fail validation?

(This seems related to ARIA-only parsing errors like lack of explicit grouping for role="radio").

Reply:
It's an Author MUST, but it's I don't think it will change the results of these tests... If you know otherwise, we should have a new file testing how that and similar required attributes affect the label... Otherwise, heading level will get text after we have a direct accessor to it. (See https://github.com/WICG/aom/issues/197 and https://github.com/WICG/aom/issues/203)